pkg/build/builder/docker.go

```diff
@@ -12,7 +12,6 @@ import (
 	dockercmd "github.com/docker/docker/builder/command"
 	"github.com/docker/docker/builder/parser"
 	docker "github.com/fsouza/go-dockerclient"
-	"github.com/golang/glog"
```

Comment:
In the package "common" has a local var glog that is the replacement.

On Sat, May 7, 2016 at 7:43 PM, Ben Parees <notifications@github.com> wrote:

> In pkg/build/builder/docker.go
> <https://github.com/openshift/origin/pull/8786#discussion_r62424311>:
>
> > @@ -12,7 +12,6 @@ import (
> >  	dockercmd "github.com/docker/docker/builder/command"
> >  	"github.com/docker/docker/builder/parser"
> >  	docker "github.com/fsouza/go-dockerclient"
> > -	"github.com/golang/glog"
>
> this import is still needed isn't it? or the util wrapper
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8786/files/9725d5171cdd040a8f619a1a7d7b332d9228209b#r62424311>
>